web/src/infra/game/GameProvider.tsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+import { GAMES_MAP } from 'games';
+import { IGameDef } from 'gamesShared/definitions/game';
+import { createContext, FC, useContext } from 'react';
+import React from 'react';
+
+const Context = createContext<GameContext>({} as GameContext);
+
+export const useCurrentGame = () => useContext(Context);
+
+export const GameProvider: FC<{ gameCode: string }> = ({ children, gameCode }) => {
+  return <Context.Provider value={{ game: GAMES_MAP[gameCode], gameCode }}>{children}</Context.Provider>;
```

Comment:
This is a great idea, a great way for us to remove the global references to GAMES_MAP currently spread out all over the code base.